src/profile/helpers/formik.ts

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from 'react';
+import { FormikProps } from 'formik';
+
+export function getIsInvalid<FormValues>(
+  formikProps: FormikProps<FormValues>,
+  fieldName: keyof FormValues
+) {
+  const isSubmitted = formikProps.submitCount > 0;
+  const isError = Boolean(formikProps.errors[fieldName]);
+
+  return isSubmitted && isError;
+}
+
+const defaultErrorRender = (error: string) => error;
+
+export function getError<FormValues>(
+  formikProps: FormikProps<FormValues>,
+  fieldName: keyof FormValues,
+  render: (error: string) => React.ReactNode = defaultErrorRender
```

Comment:
You are using `React.ReactNode` here but your import says `{ ReactNode } from 'react'`

Reply:
Fixed 👍